assert/assertion_compare.go

```diff
@@ -366,6 +366,18 @@ func Positive(t TestingT, e interface{}, msgAndArgs ...interface{}) bool {
 	return compareTwoValues(t, e, zero.Interface(), []CompareType{compareGreater}, "\"%v\" is not positive", msgAndArgs)
 }
 
+// NonPositive asserts that the specified element is positive
+//
+//    assert.Positive(t, 0)
+//    assert.Positive(t, -1.23)
+func NonPositive(t TestingT, e interface{}, msgAndArgs ...interface{}) bool {
```

Comment:
I think the functions should be called `Nonpositive()` because this is a single word in the mathematical definition for nonpositive. https://www.merriam-webster.com/dictionary/nonpositive

Reply:
Agree that `Nonpositive` would follow the mathematical naming convention (found some more discussion of it [here](https://math.stackexchange.com/questions/1138036/math-terminology-what-are-rules-regarding-hyphens-nonzero-vs-non-zero)), though my personal preference is for `NonPositive` and `NonNegative` as they're easier to read and I think that's more important than exactly adhering to the dictionary.

Not sure how others feel. I know there are standard lib Go functions that I would have re-cased, so maybe this is just me.